components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -452,9 +459,10 @@ private boolean deployBusinessRule(String nodeURL, Artifact deployableSiddhiApp,
 
     public int redeployBusinessRule(String businessRuleUUID) {
 
-        int status = TemplateManagerConstants.OPERATION_FAILED;
+        int status = TemplateManagerConstants.SAVE_SUCCESSFUL_NOT_DEPLOYED;
         BusinessRule businessRule = new QueryExecutor().executeRetrieveBusinessRule(businessRuleUUID);
         if (businessRule == null) {
+            status = TemplateManagerConstants.OPERATION_FAILED;
```

Comment:
We can directly return the value without assign it to `status`

Reply:
Fixed by 2c0119c4c48e1077512e685493252c388e68a520